TicTacToe/TicTacToe.js

```diff
@@ -1,105 +1,95 @@
 class Game {
-    moves = 0;
-    user = 'X';
-    board = [['-', '-', '-'], ['-', '-', '-'], ['-', '-', '-']];
-    messages = "Game - START!";
-    complete = false;
 
+    constructor() {
+        this.reset();
+    }
+
+    // resets the board
     reset() {
+        this.n = 3;
         this.moves = 0;
         this.user = 'X';
         this.board = [['-', '-', '-'], ['-', '-', '-'], ['-', '-', '-']];
         this.messages = "Game - START!";
         this.complete = false;
     }
 
-    getMoves() {
-        return this.moves;
-    };
-
+    // displays the board in text form
+    // don't really need this, but it's good to see
     displayBoard() {
-        let text = '';
-        for (let i = 0; i < 3; i++) {
-            for (let k = 0; k < 3; k++) {
-                text += " " + this.board[i][k];
-            }
-            text += "\n";
-        }
+        let text = this.board.join("\n").replaceAll(",", " ");
```

Comment:
Ha, this is kinda funny. I wouldn't have thought of doing it this way, but I guess stringifying arrays automatically puts spaces between them.

I guess the way I would've done it is:
```javascript
let text = this.board.map(row => row.join(" ")).join("\n");
```
but this is fine too.